src/save/common/save_slot.rs

```diff
@@ -335,7 +335,8 @@ impl Read for Regions {
         regions.unlocked_regions_count = br.read_u32()?;
 
         for _i in 0..regions.unlocked_regions_count as usize {
-            regions.unlocked_regions.push(br.read_u32()?);
+            let region = br.read_u32();
```

Comment:
This change doesn't need to exist, but allows for easier debugging.

Reply:
I don't think change that allow for easier local debugging should be included.